about.html

```diff
@@ -0,0 +1,131 @@
+                <a href="#" class="banLink">English</a>
+                <a href="#" class="banLink">My Page</a>
+                <a href="#" class="banLink">Logout</a>
+            </div>
+        
+            <nav>
+                <div class="navbar">
+                <button class="menu-btn"><img src="./assets/icons/mobile_menu 1.png" alt=""></button>
+                  <div class="navbar-logo">
+                    <a class="fade" href="./home.html"><img src="./assets/logo/svg/Color logo - no background.svg" alt="Logo"></a>
+                  </div>
+                  <ul class="menu-items">
+                    <li><a class="menuLineItems fade-link" href="./home.html">Home</a></li>
+                    <li><a class="menuLineItems" href="#">Program</a></li>
+                    <li><a class="menuLineItems" href="home.html#JoinNPConnect">Join</a></li>
+                    <li><a class="menuLineItems" href="#">Sponsor</a></li>
+                    <li><a class="menuLineItems" href="#">News</a></li>
+                    <li id="campaign"><a class="menuLineItems link-fade" href="./home.html">NPConnect</a></li>
+                  </ul>
+                </div>
+              </nav>
+
+
+          </header>
+
+      <section class="frame4">
+        <div class="heroContent">
+            <div class="aboutContentGroup">
+                <div class="theme">
+                    <h3 id="themeSub">"Hello! Adventurer."</h6>
+                    <h1 id="themeTitle">NPConnect - What's it all about?
+                    </h1>
+                </div>
+                <div class="aboutText">
+                    <h4>Welcome to D&D Hub, the ultimate online destination for Dungeons and Dragons enthusiasts! Our platform is dedicated to providing a seamless and immersive gaming experience for players and Dungeon Masters (DMs) alike. Whether you're a seasoned adventurer or a newcomer to the realms of tabletop role-playing, D&D Hub is here to support your journey.
+                    </h4>
+                </div>
+            </div>
+            <div class="frame1">
+                <h4>Please contact us per Email for any further questions about NpConnect</h3>
+                    <a href="#"><p>NPConnect@Connectinggamers.com</p></a>
+            </div>
+        </div>
+    </section>
+
+    <section class="logoSection">
+        <div class="frame3">
+            <div class="frame2">
+                <div class="frame1">
+                    <h2>NPConnect Logo</h2>
+                    <div id="ruler"></div>
+                </div>
+                <p>The logo of NPConnect decided by the founder. The logo with the shield and battle axes represents the brother and sisterhood of all players. We protect eachoter. We fight for eachother. We are a party!!</p>
+            </div>
+                <img class="logoSectionimage" src="./assets/logo/svg/Color logo with background.svg">
+        </div>
+    </section>
```

Comment:
- [ ] The middle section should also look like the given design and have the following:
- a heading
- a paragraph
- a large logo

Your design
![image](https://github.com/George7h/Module-1-capstone-project/assets/81932478/3642fe94-6b38-4b12-aac2-d7f15d0089ac)



The design should given design
![image](https://github.com/George7h/Module-1-capstone-project/assets/81932478/aaf389ae-e683-4a55-b8a6-989029e4925b)

Reply:
My design didn't do this at all as you are showing.